docs/core/install/linux-package-mixup.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Troubleshoot .NET Package Mix ups on Linux
+description: Learn about how to troubleshoot strange .NET package errors on Linux.
+author: omajid
+ms.date: 05/12/2021
+no-loc: ['usr','lib64','share','dotnet','libhostfxr.so', 'fxr', 'FrmaeworkList.xml', 'System.IO.FIleNoteFoundException']
```

Comment:
Should be `File` and `Not`, right?

Reply:
Stupid keyboard.. yeah it was the keyboards problem.. not me.